services/core-web/src/components/modalContent/ViewIncidentModal.js

```diff
@@ -257,7 +257,7 @@ export class ViewIncidentModal extends Component {
           align="left"
           pagination={false}
           columns={this.columns()}
-          locale={{ emptyText: "This incident does not contain any final documents" }}
+          locale={{ emptyText: "No Data Yet" }}
```

Comment:
Do we really need to replace empty text messages? Maybe move "No Data Yet" to constants?